src/woeusb

```diff
@@ -1365,10 +1367,11 @@ mount_target_filesystem(){
 
 	mount \
 		--options "${mount_options}" \
+		"${fstype_options}" \
 		"${target_partition}" \
 		"${target_fs_mountpoint}" \
 		|| (
-			echo_with_color red "${FUNCNAME[0]}: Error: Unable to mount target partition"
+			echo_with_color red "${FUNCNAME[0]}: Error: Unable to mount target partition\n"
```

Comment:
Would you mind explain why a newline is required?  The `echo_with_color` function should automatically output an newline in the first place...?

Reply:
Apologies, misread `printf_with_color` rather than `echo_with_color`